setup.py

```diff
@@ -121,7 +121,6 @@ def run(self):
     "pytest-cov",
     "pytest-docker>=0.7.2",
     "pytest-timeout>=1.3.3",
-    "pytest-cov>=2.6.1",
```

Comment:
I think we need to probably need to remove the one without the version specifier?